app/models/form/sales/subsections/shared_ownership_scheme.rb

```diff
@@ -43,8 +43,8 @@ def pages
       (Form::Sales::Pages::Deposit.new("deposit_shared_ownership_optional", nil, self, ownershipsch: 1, optional: true) if form.start_year_after_2024?),
       Form::Sales::Pages::DepositValueCheck.new("deposit_joint_purchase_value_check", nil, self, joint_purchase: true),
       Form::Sales::Pages::DepositValueCheck.new("deposit_value_check", nil, self, joint_purchase: false),
-      Form::Sales::Pages::DepositDiscount.new("discount", nil, self, optional: false),
-      (Form::Sales::Pages::DepositDiscount.new("discount_optional", nil, self, optional: true) if form.start_year_after_2024?),
+      Form::Sales::Pages::DepositDiscount.new("deposit_discount", nil, self, optional: false),
+      (Form::Sales::Pages::DepositDiscount.new("deposit_discount_optional", nil, self, optional: true) if form.start_year_after_2024?),
```

Comment:
We have `discount` page ID somewhere else, which messes with the routing and `deposit_discount` is more correct for this anyway!